packages/ui/src/exports/shared/index.ts

```diff
@@ -12,8 +12,8 @@ export { abortAndIgnore, handleAbortRef } from '../../utilities/abortAndIgnore.j
 export { requests } from '../../utilities/api.js'
 export { findLocaleFromCode } from '../../utilities/findLocaleFromCode.js'
 export { formatAdminURL } from '../../utilities/formatAdminURL.js'
-export { formatDate } from '../../utilities/formatDate.js'
-export { formatDocTitle } from '../../utilities/formatDocTitle.js'
+export { formatDate } from '../../utilities/formatDocTitle/formatDateTitle.js'
```

Comment:
Ideally, the function should be called "formatDateTitle" for consistency, but since it's exported, it would be a breaking change.